packages/code-du-travail-frontend/src/common/tools.js

```diff
@@ -0,0 +1,55 @@
+import { getRouteBySource, SOURCES } from "@cdt/sources";
+
+// /!\ Beware, the order of the tools here is important, don’t mess with it
+// unless you know what you are doing !
+export const tools = [
+  {
+    title: "Modèles de documents",
```

Comment:
Do you think we can import tools @cdt/data  and add only add custom items to the list ?

Reply:
That's a good question ! I would like to, but it would link the data and the frontend packages together. We could also extract the data into a package but that seems a bit like an overkill. I really really don't know.
You decide :D

Reply:
we already import data from @cdt/data for tools, it's not problem imho